api/blast_ui.api.inc

```diff
@@ -179,13 +269,16 @@ function run_BLAST_tripal_job($program, $query, $database, $output_filestub, $op
 
   // Allow administrators to use an absolute path for these commands.
   // Defaults to using $PATH.
-  $blast_path = variable_get('blast_path', '');
+  $blast_path    = variable_get('blast_path', '');
   $blast_threads = variable_get('blast_threads', 1);
-
+  
+//eksc- No,don't do this in case ".p*" appears somewhere in the db name, not just the 
+// suffix.If administrator includes the suffix, it will have to be an error, as it is for  
+// the BLAST commands themselves.
   // Strip the extension off the BLAST target
-  $database = preg_replace("/(.*)\.[pn]\w\w$/", '$1', $database);
```

Comment:
See comment about about being consistent about whether to strip the extension or validate when the blast database node is created.